modules/ROOT/pages/metrics-list.adoc

```diff
@@ -44,124 +74,141 @@ The last column of the table includes the feature or features that must be enabl
 {empty} +
 
 .Metrics that are available for Open Liberty
-[%header,cols="3,3,6,2,2"]
+[%header,cols="3,3,6,2,2,2"]
 |===
 
 |MicroProfile Metrics 3.0 name
 |MicroProfile Metrics 3.0 Prometheus name(s)
 |Type and description
+|Monitoring component
 |Feature(s) required
 |Version introduced
 
 |classloader.loadedClasses.count
 |base_classloader_loadedClasses_count
 |The number of classes that are currently loaded in the JVM.
 This metric is a gauge.
+|Base metric, no monitoring component associated. Will always be available.
```

Comment:
Rather than using this statement (which wont pass acrolinx anyway), we should explain what Base metrics are in the section before the table. Then,

Base metric, no monitoring component associated. Will always be available.
-->
Base metric